nest/02.introduction.md

```diff
@@ -0,0 +1,57 @@
+On a smartphone, the applications are usually automatically upgraded overnight when you're charging your phone, if an update is available.
+Here, it's up to the user to upgrade their installed packages whenever they want or need to.
+
+To use Nest, the only interface available for now is the Command Line Interface (CLI).
+It might be a bit nerve racking for someone who has never seen a terminal or typed any command before, but don't worry, this guide is here to help you feel comfortable!
+
+## 1.2 Packages and dependencies
+Often, a package requires another one to work properly.
+Say, for example, someone develops a messenging application, which relies on encrypted messages to provide communication between two users.
+This application will probably use a package, written by someone else, to encrypt the messages sent with their app.
+We say that such messenging application **depends** on that encryption package, because without that package, the application would probably not work at all.
+Now, if you want that application to work properly, you'll also need to install the package they used to encrypt messages.
+
+If a package is required by another one to work properly, that package is called a **dependency**.
+Whenever you install a package via a package manager (and Nest as well), it will check if the package in question has any dependencies.
+And if it does, it will install only the dependencies that are not already installed on the OS.
+So, when you install a package, don't be surprised if you see other packages getting installed too, they are just dependencies needed for that package to work properly.
+
+## 1.3 Repositories
+Packages are stored in repositories.
+Think of a repository as a big folder containing the packages.
+Within each repository, there are a certain number of categories, and a package has to belong to one, and only one of them.
+The categories are a way to distinguish packages with the same name, but having different purposes.
+This means that we can have two packages with the same name, in the same repository but not in the same category, coexisting together, without any issue.
+
+[//]: # (TODO: Add link to the section 3.1)
+For now, all you need to know is that there are official and unofficial repositories for Nest.
+Only `stable`, which is an official repository, is enabled by default.
+This repository contains all the packages that are considered stable and safe to use.
+The repositories will be more detailed in [a future section]().
+
+## 1.4 Package name syntax
+[//]: # (TODO: add link to the section 3.3)
+The packages names are represented in the following way by Nest: `repository::category/package#[version|version_req]` (*version_req* means *version requirement*).
+The concept of version requirement will be detailed in [this section]().
+If you don't specify any version or version requirement, the latest one available will be installed by default.
```

Comment:
```suggestion
If no version or version requirement is specified, the latest one available is taken by default.
```
(I think this syntax is used for other things than installing)

Reply:
(It is)